src/components/Product.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+
+function Product({ bagLabel, bagImage, bagPrice, bagTitle}) {
+    return (
+        <article>
+            <span>{bagLabel}</span>
+            <img src={bagImage} alt={bagTitle}/>
+            <p>{bagTitle}</p>
+            <h4>{bagPrice}</h4>
```

Comment:
Als je vanuit App.js de bagPrice enkel als getal doorgeeft voeg je hier de tekst € en ,- toe. Op die manier is de prijs namelijk een integer, waarmee je berekeningen zoals kortingen kunt blijven doen.

Reply:
Ja goeie!